src/cases/containers/case-hearings/case-hearings.component.ts

```diff
@@ -39,8 +37,37 @@ export class CaseHearingsComponent implements OnInit, OnDestroy {
   }
 
   public ngOnInit(): void {
-    this.upcomingHearings$ = this.getHearsListByStatus(EXUISectionStatusEnum.UPCOMING);
-    this.pastAndCancelledHearings$ = this.getHearsListByStatus(EXUISectionStatusEnum.PAST_AND_CANCELLED);
+    this.getHearsListByStatus(EXUISectionStatusEnum.UPCOMING).subscribe(hearings => {
+      if (hearings.length) {
+        const viewModels: CaseHearingViewModel[] = this.convertServiceToVM(hearings);
+
+        this.upcomingHearings$ = of(viewModels.sort((a, b) => {
+          return new Date(a.lastResponseReceivedDateTime) > new Date(b.lastResponseReceivedDateTime) ? 1 : -1;
+        }).sort((a, b) => {
+          return new Date(a.creationDateTime) > new Date(b.creationDateTime) ? 1 : -1;
+        }).sort((a) => {
+          return a.hearingListingStatus === 'WAITING TO BE LISTED' ? -1 : 1;
+        }));
+      }
+    });
+
+    this.getHearsListByStatus(EXUISectionStatusEnum.PAST_AND_CANCELLED).subscribe(hearings => {
+      if (hearings.length) {
+        const viewModels: CaseHearingViewModel[] = this.convertServiceToVM(hearings);
+        this.pastAndCancelledHearings$ = of(viewModels.sort((a, b) => {
+          return new Date(a.lastResponseReceivedDateTime) > new Date(b.lastResponseReceivedDateTime) ? 1 : -1;
+          }).sort((a) => {
+            return !a.lastResponseReceivedDateTime ? 1 : -1;
```

Comment:
we don't sort the lastResponseReceivedDateTime, but the new field:
hearingRequestDateTime

Reply:
look in the branch and I am sorting by hearingRequestDateTime